xdsl/dialects/x86/register.py

```diff
@@ -110,8 +110,8 @@ class RFLAGSRegisterType(X86RegisterType):
     name = "x86.rflags"
 
     @classmethod
-    def instruction_set_name(cls) -> str:
-        return "x86"
+    def register_set_name(cls) -> str:
+        return "x86_rflags"
```

Comment:
are these standard official names for these things? I'm not really sure what this method is used for so not sure how to review

Reply:
I have no idea if they even have official names, as far as I'm concerned the most important thing is that the names don't overlap if the registers don't overlap.

Reply:
Why wouldn't the operation name be sufficient for this?

Reply:
I couldn't remember, so let's remove it, and add it back later when we need to.